backend/article/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+URL Configuration of article
+"""
+from django.urls import path
+from article import views
+
+urlpatterns = [
+    # path('signup', views.signup, name='signup'),
```

Comment:
Is this file supposed to be discarded?

Reply:
Resolved. I mistakenly added file that I was working on.